docs/tutorials/creating-a-worker.md

```diff
@@ -0,0 +1,387 @@
+---
+description: Learn how to create a Prefect worker to run your flows.
+tags:
+    - work pools
+    - workers
+    - orchestration
+    - flow runs
+    - deployments
+    - storage
+    - infrastructure
+    - tutorial
+    - recipes
+---
+
+# Creating a Worker
```

Comment:
Any thoughts on this heading vs "Developing a worker"? I'm wondering if users who just want to start a worker or learn more about workers might end up here accidentally.

Reply:
+1 this title is a bit too vague. What about "Developing a new worker type"?